modules/ossm-observability-visual.adoc

```diff
@@ -0,0 +1,35 @@
+////
+This module is included in the following assemblies:
+- ossm-observability
+////
+
+[id="ossm-observability-visual_{context}"]
+= Visualizing your service
+
+The Kiali operator works with the telemetry data gathered in {ProductName} to provide graphs and real time network diagrams of the applications, services, and workloads in your namespace.
+
+The Overview page displays all the namespaces that have services in your mesh. You can reveal deeper insights about the data traveling through your Service mesh or help identify problems with services or workloads in your service mesh with the following graphs and visualizations. 
+
+[id="ossm-observability-topology_{context}"]
+== Namespace graphs
+
+The namespace graph is a map of the services, deployments and workflows in your namespace and arrows that show how data flows through them. To view a namespace graph: 
+
+1. Click *Graph* in the main navigation.
++
+2. Select `bookinfo` from the *Namespace* menu. 
++
+image:ossm-kiali-graph.png[Kiali graph]
+
+If your application uses version tags, like the Bookinfo sample application, you can see a Version graph select a graph from the Graph Type drop down menu. There are several graphs to choose from: 
```

Comment:
Is there some missing punctuation here?  This reads a bit choppy.

Reply:
Yes made a change